### PR TITLE
Code tidy of [pxObject] 

### PR DESCRIPTION
**Pull request by FitzerIRL**

### CODE REVIEW

File: examples/pxScene2d/src/pxScene2d.cpp [hunk 1]
```diff
@@ -1125,13 +1125,13 @@ void pxScene2d::onUpdate(double t)
...
       // TODO:  update / render times need some work...
 
-      // double draw_ms   = ( (double) sigma_draw     / (double) frameCount ) * 1000.0f; // Average frame  time
-      // double update_ms = ( (double) sigma_update   / (double) frameCount ) * 1000.0f; // Average update time
+      double draw_ms   = ( (double) sigma_draw     / (double) frameCount ) * 1000.0f; // Average frame  time
+      double update_ms = ( (double) sigma_update   / (double) frameCount ) * 1000.0f; // Average update time
 
-      // rtLogDebug("%g fps   pxObjects: %d   Draw: %g   Tex: %g   Fbo: %g     draw_ms: %0.04g   update_ms: %0.04g\n",
-      //     fps, pxObjectCount, dpf, bpf, fpf, draw_ms, update_ms );
+      rtLogError("%d fps   pxObjects: %d   Draw: %g   Tex: %g   Fbo: %g     draw_ms: %0.04g   update_ms: %0.04g\n",
```
Comment (conniefry) [inline examples/pxScene2d/src/pxScene2d.cpp]: Is this change to Error and being commented in due to local debugging, but shouldn't be in PR?

Comment (FitzerIRL) [reply examples/pxScene2d/src/pxScene2d.cpp]: The code is no longer commented - but compiled out by a #define 

It’s inactive - unless developer builds it

Comment (FitzerIRL) [reply examples/pxScene2d/src/pxScene2d.cpp]: Any additional changes/comments ?

**Comment by FitzerIRL** [comment]
Abandoned

**Pull request closed without merging**